### PR TITLE
Create WomenWhoCodeMobileSummit

### PR DESCRIPTION
**Pull request by Madonahs**

### CODE REVIEW

File: _conferences/womenwhocodemobilesummit.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Women Who Code Mobile Summit 2022"
+website: https://www.womenwhocode.com/mobile/about
```
Comment (cketti) [inline _conferences/womenwhocodemobilesummit.md]: This should be the website for the event. I suggest using the other URL as that is what allows people to register for the event.

```suggestion
website: https://hopin.com/events/women-who-code-mobile-summit/registration
```

File: _conferences/womenwhocodemobilesummit.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+location: Online
+online: true
+
+date_start: 2022-07-27
+date_end: 2022-07-27
+
+cfp:
+start:  2022-07-27
+end:    2022-07-27
+site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```
Comment (cketti) [inline _conferences/womenwhocodemobilesummit.md]: From the website it looks like speakers have already been selected and there is/was no public CfP. So you can leave out the whole `cfp` section.

File: _conferences/womenwhocodemobilesummit.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+website: https://hopin.com/events/women-who-code-mobile-summit/registration
+location: Online
+online: true
+
+date_start: 2022-07-27
+date_end: 2022-07-27
+
+start:  2022-07-27
+end:    2022-07-27
+site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```
Comment (cketti) [inline _conferences/womenwhocodemobilesummit.md]: These lines are part of the `cfp` section and should have been removed also.

```suggestion
start:  2022-07-27
end:    2022-07-27
site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```

**Review by cketti — COMMENTED** [review]
For the file name the convention we seem to have ended up with is words separated by dashes. If there's a chance the conference happens more than once, it's also a good idea to include the year.
My suggestion is to use the following file name: `wwc-mobile-summit-2022.md`

File: _conferences/womenwhocodemobilesummit.md (hunk 3 above)
Comment (Madonahs) [reply _conferences/womenwhocodemobilesummit.md]: Makes sense :) I also like the name suggestion. Thanks for reviewing.

**Review by cketti — APPROVED** [review]
Thanks for adding the event :+1:

**Pull request merged**